api/collections.py

```diff
@@ -328,6 +328,7 @@ def get(self, cid):
         for sess in sessions:
             sess['measurements'] = session_measurements.get(sess['_id'], None)
             sess['subject_code'] = sess.get('subject', {}).get('code', '') # FIXME when subject is pulled out of session
+            sess['attachment_count'] = len([f for f in sess.get('files', []) if f.get('flavor') == 'attachment'])
```

Comment:
We use it to render a paperclip for each session. The paperclip is present if there are attachments.
It's possible to move this client-side, but it is probably faster to do it here.

Note also that the attachment_count is already computed for projects, sessions and acquisitions:
https://github.com/scitran/api/blob/aloha/api/containers.py#L143